stacks-signer/src/cli.rs

```diff
@@ -106,7 +106,7 @@ pub struct SignArgs {
     pub config: PathBuf,
     /// The data to sign
     #[arg(required = false, value_parser = parse_data)]
-    pub data: Vec<u8>,
+    pub data: String,
```

Comment:
If we're gonna take this arg in as a `String`, we need to assume that it's `b58` encoded, and turn it back into a `Vec<u8>` rather than passing the string bytes around.

Reply:
Yes you are def right. Forgot about needing to add this change.